framework/src/modules/chain/api/controllers/node.js

```diff
@@ -57,8 +57,12 @@ function NodeController(scope) {
 	};
 	blockReward = new BlockReward();
 	getNetworkHeight = promisify(library.modules.peers.networkHeight);
-	getTransactionsCount = promisify(library.modules.transactions.shared.getTransactionsCount);
-	updateForgingStatus = promisify(library.modules.delegates.updateForgingStatus);
```

Comment:
Again, something is wrong with the formatting on your branch or maybe the whole feature/improve_transactions_processing_efficiency. Please solve this @mitsujutsu.

Reply:
It seems like it's happening all the time 😕